code/datums/abilities/vampire.dm

```diff
@@ -401,6 +406,8 @@
 
 			thralls += M
 
+			var/mob/living/carbon/human/M_human = victim
```

Comment:
here actually you could just change the type of `M` from `/mob/living` to `/mob/living/carbon/human` since it's already under an `ishuman` check, no idea why it isn't already.